aries-site/src/pages/extend/designer-guidance.js

```diff
@@ -0,0 +1,130 @@
+        title={title}
+        description={pageDetails.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/extend/designer-guidance"
+      />
+      <ContentSection>
+        <Subsection name={title} level={1} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+        </Subsection>
+        <Subsection name="Getting started">
+          <SubsectionText>
+            We're excited you're using the HPE Design System! Below are a set of
+            resources that will help you get started using the HPE Design System
+            components in your designs. If you don't find what you're looking
+            for, please reach out in the #hpe-design-system channel on{' '}
+            <Anchor
+              label="Slack"
+              href="https://grommet.slack.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            />
+            .
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Setting up your Figma account" level={3}>
+          <SubsectionText>
+            If you are new to Figma, the{' '}
+            <Anchor
+              label="Figma Getting Started"
+              href="https://help.figma.com/hc/en-us/categories/360002051613-Getting-Started"
+              target="_blank"
+              rel="noopener noreferrer"
+            />{' '}
+            page is a good place to start.
+          </SubsectionText>
+          <SubsectionText>
+            Create your account with your HPE email to be able to have access to
+            all of the HPE Design System files.
+          </SubsectionText>
+          <SubsectionText>
+            If you would prefer to access the HPE Design System library in a
+            desktop app, you can download the{' '}
+            <Anchor
+              label="Figma Desktop App"
+              href="https://help.figma.com/hc/en-us/articles/360039823654-Download-the-Figma-Desktop-App#Download_the_Desktop_App"
+              target="_blank"
+              rel="noopener noreferrer"
+            />{' '}
+            for MacOS or Windows.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Joining the HPE Design System Figma team" level={3}>
+          <SubsectionText>
+            Once you have made a Figma account with your HPE email, you can
+            request to join the HPE Design System team.
+          </SubsectionText>
+          <SubsectionText>
+            1) Log into Figma and look for an HPE icon in your left control bar.
+            Navigate to this section, you will see all the HPE Figma teams. Look
+            for "HPE Design System" and click "Request to join".
+          </SubsectionText>
+          <SubsectionText>
+            2) Click on HPE. Here, you will see all HPE Figma teams.
+          </SubsectionText>
+          <SubsectionText>
+            3) Look for "HPE Design System" and click "Request to join".
+          </SubsectionText>
```

Comment:
Semantically, this should probably be an ordered list.

Reply:
Yes, I think you're right

Reply:
Done : )